ui/pip_button.js

```diff
@@ -122,6 +122,10 @@ shaka.ui.PipButton = class extends shaka.ui.Element {
    */
   async onPipClick_() {
     try {
+      if ('documentPictureInPicture' in window) {
```

Comment:
I think this would be more correct:
```
      if ('documentPictureInPicture' in window) {
         // If you were fullscreen, leave fullscreen first.
         if (document.fullscreenElement) {
           document.exitFullscreen();
         }
         await this.toggleDocumentPictureInPicture_();
       } else if (!document.pictureInPictureElement) {
```

Reply:
@avelad  There's no need to call `exitFullscreen()` because Doc PiP will open a new window, triggering fullscreen to exit itself automatically.

Reply:
and the transition is smooth?

Reply:
Yup! Feel free to give it a try locally.